change/@microsoft-teams-js-e551f18c-3f83-46d2-9be7-9f56341f85df.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Removed testing api version number and corrected faulty tag on API",
```

Since these are patch notes for external devs, what about:

```suggestion
  "comment": "Fixed bug where `authentication.getUser` telemetry was being incorrectly recorded",
```

---

Applied suggestion! Thank you!